contracts/SignerBondsUniswapV2.sol

```diff
@@ -77,6 +77,9 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
     address public immutable assetPool;
     address public immutable collateralToken;
 
+    // Currently approved operators that can swap signer bonds on Uniswap V2
+    mapping(address => bool) public approvedOperators;
```

Comment:
I would consider naming them differently than operators given that "operator" is a very specific role in Keep Network and contracts. Maybe `approvedSwappers`?

Reply:
I changed the name of mapping to `approvedSwappers` in https://github.com/keep-network/coverage-pools/pull/148/commits/cee62f0dba2afa4442e29eff73ffec2170ac3dd2